### PR TITLE
Add GlossaryDocument & GlossaryRecord

### PR DESCRIPTION
**Pull request by GoddessEyes**
* rework db-connection, app-instance for test
* add main.py for `fastapi dev` command
* bump fastapi to 0.111
* rework settings instance

### CODE REVIEW

File: backend/tests/conftest.py [hunk 1]
```diff
@@ -1,48 +1,59 @@
...
+from sqlalchemy.orm import Session, sessionmaker
 
-from app import create_app, db, models, schema
-from app.db import get_db, init_connection
+from app import models, schema
+from app.db import Base, get_db
+from main import app
 
-# pylint: disable=C0116
+SQLALCHEMY_DATABASE_URL = "sqlite:///"
```
Comment (ArchiDevil) [inline backend/tests/conftest.py:10]: Where is the actual file be located with this connection URI?

File: backend/alembic/versions/6d107741a92e_add_glossary.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('glossary_record',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('created_at', sa.DateTime(), nullable=False),
+    sa.Column('updated_at', sa.DateTime(), nullable=False),
+    sa.Column('document_id', sa.Integer(), nullable=False),
+    sa.Column('comment', sa.String(), nullable=False),
+    sa.Column('en_term', sa.String(), nullable=False),
+    sa.Column('ru_term', sa.String(), nullable=False),
```
Comment (ArchiDevil) [inline backend/alembic/versions/6d107741a92e_add_glossary.py:40]: It should be src and dst terms to be consistent with other parts of DB

File: backend/alembic/versions/6d107741a92e_add_glossary.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+# revision identifiers, used by Alembic.
+revision: str = '6d107741a92e'
+down_revision: Union[str, None] = 'b3e764c93fac'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
```
Comment (ArchiDevil) [inline backend/alembic/versions/6d107741a92e_add_glossary.py:24]: Remove these comments as you've already adjusted the code

File: backend/alembic/versions/6d107741a92e_add_glossary.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    op.create_table('glossary_document',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('created_by', sa.Integer(), nullable=False),
+    sa.Column('created_at', sa.DateTime(), nullable=False),
+    sa.Column('updated_at', sa.DateTime(), nullable=False),
+    sa.ForeignKeyConstraint(['created_by'], ['user.id'], ),
+    sa.PrimaryKeyConstraint('id')
```
Comment (ArchiDevil) [inline backend/alembic/versions/6d107741a92e_add_glossary.py:31]: This needs to be formatted

File: backend/app/glossary/models.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from app.db import Base
+from app.schema import User
+
+if TYPE_CHECKING:
+    from app.schema import User
+
+
+class GlossaryDocument(Base):
```
Comment (ArchiDevil) [inline backend/app/glossary/models.py:14]: Is it a good idea to spread database schema parts all the over the source tree? It was convenient to have everything inside schema.py

Comment (GoddessEyes) [reply backend/app/glossary/models.py:14]: It's even worse to store everything in one place. One day the schema file will become unreadable
Project needs directory hierarchy (architecture)

File: backend/alembic/versions/6d107741a92e_add_glossary.py (hunk 2 above)
Comment (GoddessEyes) [reply backend/alembic/versions/6d107741a92e_add_glossary.py]: Done

File: backend/alembic/versions/6d107741a92e_add_glossary.py (hunk 3 above)
Comment (GoddessEyes) [reply backend/alembic/versions/6d107741a92e_add_glossary.py]: Done

File: backend/alembic/versions/6d107741a92e_add_glossary.py (hunk 4 above)
Comment (GoddessEyes) [reply backend/alembic/versions/6d107741a92e_add_glossary.py]: Done

File: backend/app/glossary/models.py (hunk 5 above)
Comment (ArchiDevil) [reply backend/app/glossary/models.py:14]: Okay, I have no objections on it, just little note: can you use `schema.py` for DB tables? It was named "schema" for DB and "models" for pydantic models and I wanted to save this naming in all parts of the code.

**Review by ArchiDevil — APPROVED** [review]

**Pull request merged**